etc/ProdOfflineConfiguration.py

```diff
@@ -1063,6 +1067,17 @@
                alca_producers=["EcalCalPi0Calib", "EcalCalEtaCalib"],
                scenario=ppScenario)
 
+DATASETS = ["AlCaHcalIsoTrk"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=False,
```

Comment:
does this switch off the alreco step as well? In that case it needs to be `True`.

I thought that the alcareco step is not toggled by this, eg. below there is:

https://github.com/dmwm/T0/blob/8b964544f540c85dd8215b5659d5f82e86c4251a/etc/ProdOfflineConfiguration.py#L1067

(but now checking in DAS, it seems that these PDs are not produced anymore).
@LinaresToine  @jeyserma